### ISSUE TITLE
Question about experiment config file

### ISSUE THREAD

**Issue by ProphecyQAQ** [Thinklab-SJTU/ThinkMatch#66]
Hello!
I am very interesting in your paper "Deep Learning of Partial Graph Matching via Differentiable Top-K". And I have a question about it. 

In willow object with random outliers experiment, bbgm's mean f1 is 69.7. But I can't reproduce it myself. Could you release your config file? 

Thank you!

**Comment by ziao-guo** [comment]
Thank you for your interest in our work and for taking the time to raise the issue!

Here is the complete configuration:

MODEL_NAME: vgg16_bbgm
DATASET_NAME: willow-all

DATASET_FULL_NAME: WillowObject

MODULE: models.BBGM.model

BACKBONE: VGG16_bn_final

BATCH_SIZE: 8
DATALOADER_NUM: 2
FP16: False

RANDOM_SEED: 123

STATISTIC_STEP: 100

GPUS:
  - 0

PROBLEM:
  TYPE: 2GM
  RESCALE:
    - 256
    - 256
  FILTER: 'unfiltered'

GRAPH:
  SRC_GRAPH_CONSTRUCT: tri
  TGT_GRAPH_CONSTRUCT: tri
  SYM_ADJACENCY: True

WillowObject:
  TRAIN_NUM: 20 
  SPLIT_OFFSET: 0
  RAND_OUTLIER: 10

TRAIN:
  START_EPOCH: 0
  NUM_EPOCHS: 10

  LOSS_FUNC: hamming

  OPTIMIZER: Adam

  LR: 4.e-3
  SEPARATE_BACKBONE_LR: True
  BACKBONE_LR: 4.e-5
  MOMENTUM: 0.9
  LR_DECAY: 0.5
  LR_STEP:  # (in epochs)
    - 1

  EPOCH_ITERS: 100  # iterations per epoch

  CLASS: none

EVAL:
  EPOCH: 2  # epoch to be tested
  SAMPLES: 100  # number of tested pairs for each class

BBGM:
  FEATURE_CHANNEL: 1024
  SOLVER_NAME: LPMP
  LAMBDA_VAL: 80.0
  SOLVER_PARAMS:
    timeout: 1000
    primalComputationInterval: 10
    maxIter: 100

Also, please note that even with a fixed random seed, we can not completely avoid randomness, so the experimental result may still exhibit a certain degree of fluctuation.

**Comment by ProphecyQAQ** [comment]
> Thank you for your interest in our work and for taking the time to raise the issue!
> 
> Here is the complete configuration:
> 
> MODEL_NAME: vgg16_bbgm DATASET_NAME: willow-all
> 
> DATASET_FULL_NAME: WillowObject
> 
> MODULE: models.BBGM.model
> 
> BACKBONE: VGG16_bn_final
> 
> BATCH_SIZE: 8 DATALOADER_NUM: 2 FP16: False
> 
> RANDOM_SEED: 123
> 
> STATISTIC_STEP: 100
> 
> GPUS:
> 
>     * 0
> 
> 
> PROBLEM: TYPE: 2GM RESCALE: - 256 - 256 FILTER: 'unfiltered'
> 
> GRAPH: SRC_GRAPH_CONSTRUCT: tri TGT_GRAPH_CONSTRUCT: tri SYM_ADJACENCY: True
> 
> WillowObject: TRAIN_NUM: 20 SPLIT_OFFSET: 0 RAND_OUTLIER: 10
> 
> TRAIN: START_EPOCH: 0 NUM_EPOCHS: 10
> 
> LOSS_FUNC: hamming
> 
> OPTIMIZER: Adam
> 
> LR: 4.e-3 SEPARATE_BACKBONE_LR: True BACKBONE_LR: 4.e-5 MOMENTUM: 0.9 LR_DECAY: 0.5 LR_STEP: # (in epochs) - 1
> 
> EPOCH_ITERS: 100 # iterations per epoch
> 
> CLASS: none
> 
> EVAL: EPOCH: 2 # epoch to be tested SAMPLES: 100 # number of tested pairs for each class
> 
> BBGM: FEATURE_CHANNEL: 1024 SOLVER_NAME: LPMP LAMBDA_VAL: 80.0 SOLVER_PARAMS: timeout: 1000 primalComputationInterval: 10 maxIter: 100
> 
> Also, please note that even with a fixed random seed, we can not completely avoid randomness, so the experimental result may still exhibit a certain degree of fluctuation.

Thank you! This would really help me a lot.
